react-accordion-component/src/index.jsx

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import Accordion from './accordion';
+
```

Comment:
Here you want to create an array of objects that contains the accordion contents. Something like:

```JSX
const topics = [
  {
    id: 'html',
    title: 'Hypertext Markup Language',
    content: 'Paragraph about HTML goes here'
  }
  // Create objects for all topics
];